src/rate-limiter/composer.json

```diff
@@ -17,7 +17,7 @@
         "symfony/polyfill-mbstring": "^1.0"
     },
     "require-dev": {
-        "symfony/cache": "^5.1||^6.0"
+        "symfony/cache": "^5.4||^6.3||^7.0"
```

Comment:
this is unecessary, dev dependencies does not need to be tested across all versions as they are not exposed to end users, I think I would use just 6.4 as the latest LTS.